src/state/reducers/visualization/visualizationSlice.ts

```diff
@@ -126,7 +126,7 @@ const initialState: VisualizationState = (() => {
     tile_pix_width: 0,
     tile_pix_height: 0,
     max_zoom: 0,
-    max_num_text: 200,
+    max_num_text: 400, // number of texts to show up
     zoom_data: zoom_data as ZoomData,
```

Comment:
What does this affect and why was this changed?

Reply:
they had set the limit to 200 labels which was preventing the labels to show up as they were more than 200 so, i have increased the limit to 400 labels and it will be able to show labels upto 400.

Reply:
These are our categories? Is this the number of rendered categories, or a hard limit on how many categories we can have? If we have 600 categories, does this mean the last 200 will never render?

Reply:
Its like a hard limit when it exceeds the limit it will not be able to show anything.

Reply:
Has this been discussed with @pranalis18 ?

Reply:
no